libs/wingc/src/jsify.rs

```diff
@@ -325,15 +325,14 @@ impl<'a> JSifier<'a> {
 					None
 				};
 
-				if !is_resource || fqn.is_none() {
-					format!("new {}({})", ctor, args)
-				} else {
-					let fqn = fqn.expect("expecting fqn to be defined");
+				if let (true, Some(fqn)) = (is_resource, fqn) {
```

Comment:
Is this related to this PR? What's the fix here?

Reply:
Just a refactoring. The way it's currently written, the second branch has to call "expect" to unwrap the FQN even though logically we _know_ the fqn is a `Some(String)`.